<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChonyaEdit</class>
 <widget class="QMainWindow" name="ChonyaEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EditButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(170, 170, 255);
	border-radius: 20px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(170, 85, 255);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>修改参数</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/change.png</normaloff>:/images/change.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品料号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>56</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LifeNum">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>35</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具编号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>265</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>材料规格:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutNum_2">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>385</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具寿命冲次数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>325</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>管制标准:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoadingLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductStand">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入材料规格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MoldID">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>请输入模具编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutNum_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="ControlStand">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>306</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <item>
     <property name="text">
      <string>NULL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格＜0.3mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格≧0.3且﹤0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格≧0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不锈钢材料厚度规格＜0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不锈钢材料厚度规格≧0.5mm</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>385</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品出数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品名称:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProductName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>95</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品名称</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProductID">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品料号</string>
    </property>
   </widget>
   <widget class="QLabel" name="Back1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>446</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>465</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增加机台:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MachineNum">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>1#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>#6</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MachineComb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>515</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>添加机台</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Back2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>445</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 175, 200);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>515</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>删除机台</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Label10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>465</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除机台:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>200</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/logo200x53.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="NameButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>515</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>删除员工</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Label10_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>465</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除员工:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NameComb">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>Back2</zorder>
   <zorder>Back1</zorder>
   <zorder>EditButton</zorder>
   <zorder>Label2</zorder>
   <zorder>Label7</zorder>
   <zorder>LifeNum</zorder>
   <zorder>Label5</zorder>
   <zorder>Label3</zorder>
   <zorder>OutNum_2</zorder>
   <zorder>Label8</zorder>
   <zorder>Label4</zorder>
   <zorder>LoadingLabel</zorder>
   <zorder>ProductStand</zorder>
   <zorder>MoldID</zorder>
   <zorder>OutNum_1</zorder>
   <zorder>ControlStand</zorder>
   <zorder>Label6</zorder>
   <zorder>Label1</zorder>
   <zorder>ProductName</zorder>
   <zorder>ProductID</zorder>
   <zorder>Label9</zorder>
   <zorder>MachineNum</zorder>
   <zorder>MachineComb</zorder>
   <zorder>AddButton</zorder>
   <zorder>DeleButton</zorder>
   <zorder>Label10</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>NameButton</zorder>
   <zorder>Label10_2</zorder>
   <zorder>NameComb</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>MoldID</tabstop>
  <tabstop>ProductName</tabstop>
  <tabstop>ProductID</tabstop>
  <tabstop>ProductStand</tabstop>
  <tabstop>ControlStand</tabstop>
  <tabstop>OutNum_1</tabstop>
  <tabstop>OutNum_2</tabstop>
  <tabstop>LifeNum</tabstop>
  <tabstop>EditButton</tabstop>
  <tabstop>MachineComb</tabstop>
  <tabstop>MachineNum</tabstop>
  <tabstop>NameComb</tabstop>
  <tabstop>DeleButton</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>NameButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
